src/wp-admin/includes/class-wp-privacy-requests-table.php

```diff
@@ -447,14 +447,18 @@ public function column_status( $item ) {
 				break;
 		}
 
-		echo '<span class="status-label status-' . esc_attr( $status ) . '">';
-		echo esc_html( $status_object->label );
+		$html = '';
```

Comment:
Just a thought: would it be an idea to have an abstract `WP_List_Table_TestCase` class which all List_Table test classes should extend and to add generic tests to that class (which would then be run for all concrete test classes) to verify that the return type of abstract method implementations complies with the expected return type (or has expected output) ?

I imagine, this might also be a good idea for the Widget classes.